tools/shared/pragma.c

```diff
@@ -658,7 +658,59 @@ do_sw(void)
           break;
         }
         LCASE(ctok);
-        if (strcmp(ctok, "always") == 0) {
+        if (strcmp(ctok, "vectorlength") == 0) {
+          assn( DIR_OFFSET(currdir, x[235]), 0);
+          if (gtok() != T_LP){
+            return true;
```

Comment:
Shouldn't this also be a warning, like for the other invalid input? Should we return here or break, like for the other cases?

It should also have a test also.